src/app/App.tsx

```diff
@@ -30,107 +64,134 @@ export default function App() {
     { id: '1h,1h', title: '1H,1H', content: 'Hello, World! [c]' },
     { id: '1h,13c', title: '1H,13C', content: 'Hello, World! [d]' },
   ];
-
   return (
-    <RootLayout>
-      <div style={{ height: '100%', display: 'flex', flexDirection: 'column' }}>
-        <div style={{ display: 'flex', flexDirection: 'column' }}>
-          <Header>
-            <Toolbar orientation="horizontal">
-              <Toolbar.Item
-                titleOrientation="horizontal"
-                id="logo"
-                title="Logo"
-              >
-                <FaMeteor />
-              </Toolbar.Item>
-            </Toolbar>
-            <Toolbar orientation="horizontal">
-              <Toolbar.Item id="a" title="User manual">
-                <FaBook />
-              </Toolbar.Item>
-              <Toolbar.Item id="b" title="General settings">
-                <FaCogs />
-              </Toolbar.Item>
-              <Toolbar.Item id="c" title="Full screen">
-                <FaTabletAlt />
-              </Toolbar.Item>
-            </Toolbar>
-          </Header>
-        </div>
+    <DropZoneContainer
+      onDrop={(files) => {
+        void loadFiles(files, dispatch);
+      }}
+    >
```

Comment:
Can you put this inside `RootLayout`? Otherwise it's not part of the shadow dom.